lib/v2/utils/read_times_tamp.dart

```diff
@@ -6,11 +6,13 @@ import 'package:timeago/timeago.dart' as timeago;
 //Link to timeago library
 //https://pub.dev/documentation/timeago/latest/timeago/timeago-library.html
 
-String timesTampToTimeAgo(String timestamp) {
-  // Note: Timestamps always come as UTC but don't have the time zone set on EOSIO
-  final covertTimesTamp = DateTime.parse("${timestamp}Z");
-
-  final timeAgo = timeago.format(covertTimesTamp, locale: 'en');
+String timesTampToTimeAgo(DateTime timestamp) {
+  final timeAgo = timeago.format(timestamp, locale: 'en');
 
   return timeAgo;
 }
+
+DateTime parseTimestamp(String timestamp) {
```

Comment:
can we ask Json or gabriel to test these date changes pls. 
@gguij004

Reply:
I checked and they're currently only displayed in "timeAgo" fashion, meaning they are shown relative, meaning time zone doesn't matter. 

Before they were stored as string and always in UTC time zone - now they're also in UTC time zone. If we ever show them in another place in the UX, we will have to use toLocal() on it - but I think that will be pretty obvious, and I didn't add this anywhere since it's not needed right now (and was UTC already anyway)